components/SentenTree/index.js

```diff
@@ -0,0 +1,57 @@
+import VisComponent from '../../VisComponent';
+import { SentenTreeBuilder,
+         SentenTreeVis } from 'sententree/dist/SentenTree';
+
+export default class SentenTree extends VisComponent {
+  static get options () {
+    return [
+      {
+        name: 'data',
+        description: 'The data table.',
+        type: 'table',
+        format: 'objectlist'
+      },
+      {
+        name: 'id',
+        description: 'The field containing the identifier of each row.',
+        type: 'string',
+        domain: {
+          mode: 'field',
+          from: 'data',
+          fieldTypes: ['string', 'integer', 'number']
+        }
+      },
+      {
+        name: 'text',
+        description: 'The field containing the text sample.',
+        domain: {
+          mode: 'field',
+          from: 'data',
+          fieldTypes: ['string']
+        }
+      },
+      {
+        name: 'count',
+        description: 'The field containing the count for each text sample.',
+        type: 'string',
+        domain: {
+          mode: 'field',
+          from: 'data',
+          fieldTypes: ['integer']
+        }
+      }
+    ];
+  }
+
+  constructor (el, {data, graphs = 3}) {
```

Comment:
It seems we are not using the `id`, `text`, and `count` parameters here. I assume we would transform the data to put those fields in standard places that SentenTree expects before initializing the visualization. It would also be great to put in defaults (set an `id` to a unique number 0 to N if no `id` is provided, set all counts to 1 if no `count` is provided). That way all that would be needed is a table that has a text column.

Also, the `graphs` parameter should be described as an option so there can be auto UI generation for setting the number of graphs to show.